client/src/game/ui/notes/NoteList.vue

```diff
@@ -45,6 +46,22 @@ const shapeName = computed(() => {
 //     searchBar.value?.focus();
 // });
 
+function handleClickOutsideDialog(event: MouseEvent): void {
+    if (searchOptionsDialog.value) {
+        if (showSearchFilters.value && !searchOptionsDialog.value.contains(event.target as Node)) {
+            showSearchFilters.value = false;
+        }
+    }
+}
+
+onMounted(() => {
+    document.addEventListener('mousedown', handleClickOutsideDialog);
```

Comment:
I know it's not present everywhere in PA, but I would change this to be `pointerdown`. It's a small change, but then it will work on both touch and mouse devices in general.

Reply:
Ah makes sense. Will do